packages/driver/cypress/e2e/commands/actions/type_special_chars.cy.js

```diff
@@ -999,7 +1003,10 @@ describe('src/cy/commands/actions/type - #type special chars', { browser: '!webk
       cy.get('input[type="number"]:first')
       .invoke('val', '12.34')
       .type('{uparrow}{uparrow}')
-      .should('have.value', '14')
+      // WebKit does not round to the step value when calling stepUp/stepDown,
+      // as we do here for the ArrowUp handler.
+      // https://bugs.webkit.org/show_bug.cgi?id=244206
+      .should('have.value', Cypress.isBrowser('webkit') ? '14.34' : '14')
```

Comment:
The bug I logged is getting a little traction, but I'm not sure how quick these things get turned around. Do we think it's worth adding our own logic here for WebKit to bring it to parity, or just documenting the issue as a current limitation?

Reply:
Option 3 ... Fix WebKt ourselves! But we'd still need something in the (probably long) interim. 

If this is a WebKit bug, what you've done make sense to me - if a user runs a test expecting 14, it *should* fail - since it will fail in production for real users, too.

Reply:
> if a user runs a test expecting 14, it should fail - since it will fail in production for real users, too

The behavior we're simulating here is a user tapping the up/down arrows inside the input, which _does_ work appropriately in WebKit. The problem is that the `stepUp`/`stepDown` functions we call to simulate those presses don't work the same way. So this would definitely need to be a documented limitation if we left things as-is.

I did write up the algorithm to handle the step logic ourselves. It's not too intensive, but it'll need a new suite of tests associated to it. I want to make sure we have other critical issues wrapped up before spending much more time on it (unless we consider this critical?).

Reply:
The cross-section of users who use `.type('{uparrow}')` on a number input who will be using experimental WebKit is probably pretty small, I'd agree that focus should be put elsewhere. How large is the change to fix this?

Reply:
The algorithm isn't very complicated, there are just a lot of bound checks/edge cases that I'd want to write thorough tests for. I'd hate to write a custom handler that introduces yet more bugs :bug:

I think we could wrap it up prior to the initial release if we start looking good on the other issues.

Reply:
Sounds good, I say the sooner we ship this, the better - little fixes like this will come up organically, if we don;'t get to them (and we might have other edges cases we don't even know about yet).